src/In.ProjectEKA.HipService/MessagingQueue/MessagingQueueManager.cs

```diff
@@ -9,37 +9,37 @@ namespace In.ProjectEKA.HipService.MessagingQueue
 
     public class MessagingQueueManager : IMessagingQueueManager
     {
-        private readonly DefaultObjectPool<IModel> objectPool;  
-        
-        public MessagingQueueManager(IPooledObjectPolicy<IModel> objectPolicy)  
-        {  
-            objectPool = new DefaultObjectPool<IModel>(objectPolicy, Environment.ProcessorCount * 2);  
+        private readonly DefaultObjectPool<IModel> objectPool;
+
+        public MessagingQueueManager(IPooledObjectPolicy<IModel> objectPolicy)
+        {
+            objectPool = new DefaultObjectPool<IModel>(objectPolicy, Environment.ProcessorCount * 2);
         }
-        
+
         public void Publish<T>(T message, string exchangeName, string exchangeType, string routeKey) where T : class
         {
-            if (message == null) { return; }
+            if (message == null) return;
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6033776)